src/go/k8s/tests/e2e/produce-consume/01-create-topic.yaml

```diff
@@ -17,8 +17,8 @@ spec:
             - /bin/bash
             - -c
           args:
-            - rpk topic create test --brokers cluster-sample-0.cluster-sample.$POD_NAMESPACE.svc.cluster.local:9092 -v
```

Comment:
This makes the timing a bit sensitive, as it can run before all three brokers are online.  
```
$ cat `find ./ -name create-test*`
2021-09-17T11:31:01.1626548Z stdout F TOPIC  STATUS
2021-09-17T11:31:01.162679779Z stdout F test   OK
2021-09-17T11:27:40.233001354Z stdout F TOPIC  STATUS
2021-09-17T11:27:40.233057262Z stdout F test   OK
2021-09-17T11:28:16.875620979Z stdout F TOPIC  STATUS
2021-09-17T11:28:16.875640454Z stdout F test   INVALID_REPLICATION_FACTOR: Replication factor is below 1 or larger than the number of available brokers.
```

I don't understand why it runs on the workers rather than on the controller though (not very familiar with these `kind` tests).

Reply:
Ah, my grep was including output from other tests.  I think the output from this test alone is just 
```
2021-09-17T11:28:16.875620979Z stdout F TOPIC  STATUS
2021-09-17T11:28:16.875640454Z stdout F test   INVALID_REPLICATION_FACTOR: Replication factor is below 1 or larger than the number of available brokers.
```

Reply:
You are suggesting to add new step in the test that will wait for cluster to form the quorum. I like this idea. My intention here was do not create partition that can be on different node. If I recall redpanda need to commit the new topic to raft0 group and then it returns to the user. If topic replication equals number of nodes I assume this means Redpanda successfully form 3 node cluster.